nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/xml/XMLReader.java

```diff
@@ -106,12 +107,27 @@ public class XMLReader extends SchemaRegistryService implements RecordReaderFact
             .required(false)
             .build();
 
+    public static final PropertyDescriptor PARSE_XML_ATTRIBUTES = new PropertyDescriptor.Builder()
+            .name("parse_xml_attributes")

```

Comment:
I think generally dashes are used in property names instead of underscores.

Reply:
That is correct. However, in this class underscores were used in the other property names, and I wanted it to be consistent with that.